index.html

```diff
@@ -8,11 +9,20 @@
     <link rel="stylesheet" href="style.css">
     <title>Jubaer Maruf</title>
 </head>
+
 <body>
     <header class="header">
         <a href="#" class="logo">My Logo</a>
         <nav class="nav">
-            
+<<<<<<< HEAD
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+            </ul>
+=======
+
+>>>>>>> cba244d1df191c9275d7cd0cf679d098ba3c0f35
```

Comment:
- [ ] Kindly remove this conflict in your code to make it professional, as these conflicting are influencing your webpage. 

| Here | and Here |
| :---: | :---: |
| ![Capture](https://user-images.githubusercontent.com/80458199/229830864-bff3a225-f6f1-4ad5-9d19-12608adb5b17.PNG) | ![Capture](https://user-images.githubusercontent.com/80458199/229831161-6a5f94e4-5527-4d7e-baf8-7c8ddf966387.PNG) |